xml/System.Diagnostics/ProcessStartInfo.xml

```diff
@@ -480,9 +482,14 @@ If you use this property to set command-line arguments, <xref:System.Diagnostics
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets a value indicating whether to start the process in a new process group.</summary>
+        <value>
+          <see langword="true" /> if the process should be started in a new process group; otherwise, <see langword="false" />. The default is <see langword="false" />.</value>
```

Comment:
```suggestion
          <see langword="true" /> if the process starts in a new process group; otherwise, <see langword="false" />. The default is <see langword="false" />.</value>
```